packages/utils/tailwind-plugins/animations/index.js

```diff
@@ -137,12 +137,22 @@ module.exports = plugin.withOptions(
             '0%': { transform: 'translateX(0)' },
             '100%': { transform: 'translateX(-100px)' },
           },
+          indeterminateBar: {
+            '0%': {
+              transform: 'translateX(0%) scaleX(0.2)',
+            },
+            '100%': {
+              transform: 'translateX(100%) scaleX(1)',
+            },
+          },
         },
         animation: {
           'slide-top': 'slideTop 0.5s ease-in-out',
           'slide-right': 'slideRight 0.5s ease-in-out',
           'slide-bottom': 'slideBottom 0.5s ease-in-out',
           'slide-left': 'slideLeft 0.5s ease-in-out',
+          'indeterminate-bar':
+            'indeterminateBar 1.5s cubic-bezier(0.65, 0.815, 0.735, 0.395) infinite normal none running',
```

Comment:
👀 ⚠️

Reply:
for me better have it style inline than create a token that affects the whole DS and on the other package.

**S** - O - L - I - D

Reply:
?

Reply:
I agree with you @andresin87 , but I think we're kind of stuck here, imho.
Since we can't define a keyframe animation inline, and if we define it in its own CSS file, the consumers of the `Progress` component will have to download that CSS file as well 🫤

Reply:
But maybe for cases like this, we could use a prefix, similar to the one used for utility classes such as `u-current-font-size` and `u-shadow-border-transition`

For example, something like `isolated-` or `standalone-` 🤷 
This would allow me to filter these out inside our TCV

Reply:
Which name would you prefer? I'm fine with either of them.